snmp/datadog_checks/snmp/data/default_profiles/_generic-entity-sensor-mib.yaml

```diff
@@ -0,0 +1,97 @@
+metrics:
```

Comment:
`_generic-entity-sensor-mib.yaml` -> `_generic-entity-sensor.yaml`

For consistency with existing `_generic-if.yaml` and others.